app/forms/mappings/ContactDetailsMapping.scala

```diff
@@ -22,12 +22,14 @@ import play.api.data.Forms._
 object ContactDetailsMapping extends Mappings {
 
   val contactDetailsMapping = mapping(
-    "name" -> text().verifying(maxLength(35, "contactDetails.name.invalid")),
-    "companyName" -> text().verifying(maxLength(35, "contactDetails.companyName.invalid")),
-    "phoneNumber" -> text()
+    "name" -> text("contactDetails.name.missing")
+      .verifying(maxLength(35, "contactDetails.name.invalid")),
+    "companyName" -> text("contactDetails.companyName.missing")
+      .verifying(maxLength(35, "contactDetails.companyName.invalid")),
+    "phoneNumber" -> text("contactDetails.phoneNumber.missing")
```

Comment:
tbh I was more readable before

Reply:
??? because of the new line ???

Reply:
But it just matches the style below where there are multiple chained calls to `verifying` and as the 'text' method call is now taking a parameter the line length would be too long, no?